lighthouse/config/defaults.py

```diff
@@ -11,6 +11,9 @@
 BARACODA_URL = f"{LOCALHOST}:5000"
 DOWNLOAD_REPORTS_URL = f"http://{LOCALHOST}:5000/reports"
 LABWHERE_URL = f"{LOCALHOST}:3010"
+LABWHERE_DESTROYED_BARCODE = os.environ.get(
```

Comment:
This is here, rather than in the constant file, as it changes between environments.